webhook/webhook.groovy

```diff
@@ -278,6 +316,69 @@ class ResponseFormatter {
     }
 }
 
+/**
+ * Spinnaker formatter
+ */
+class SpinnakerFormatter {
+    def format(String event, JsonBuilder data) {
+        def eventTypeMetadta = Globals.eventToSupported(event)
```

Comment:
Typo

Reply:
@DarthFennec, thanks for pointing out this. I will fix this and update the pull request.

Reply:
Fixed the typo